src/components/Card.css

```
@@ -26,7 +37,7 @@ p {
 
 .link-card:is(:hover, :focus-within) {
     background-position: 0;
-    background: var(--background);
+    background: var(--background-high-contrast);
```

Stylelint (17.14.0)

[error] Overridden property "background-position" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)